app/docs/0.14.x/configuration.md

```diff
@@ -128,12 +256,15 @@ two files into the prefix directory, which looks like so:
 ```
 /usr/local/kong
 ├── nginx-kong.conf
-├── nginx.conf
+└── nginx.conf
 ```
 
-If you wish to include other `server` blocks in your Nginx configuration, or if
-you must tweak global settings that are not exposed by the Kong configuration,
-here is a suggestion:
+If you must tweak global settings that are defined by Kong and not exposed by
+the Kong configuration, here is a suggestion, or even customize the Kong Nginx
+sub-configuration file to, eventually, add other Lua handlers or customize the
+`lua_*` directives, you can inline the contents of the `nginx_kong.lua`
+configuration template into a custom template file (in this example called
+`custom_nginx.template`) like this:
```

Comment:
> If you must tweak global settings that are defined by Kong and not exposed by the Kong configuration

This should probably be updated to reflect the new "rare conditions" that warrant a custom template (that you perfectly described above)

> here is a suggestion, or even customize the Kong Nginx sub-configuration file to, eventually, add other Lua handler

Something looks wrong here?

Reply:
> Something looks wrong here?

Good catch, an old sentence fragment was incorrectly pasted.